src/extensions/renderer/base/coord-ele-math/labels.js

```diff
@@ -362,8 +362,7 @@ BRp.getLabelText = function( ele, prefix ){
     let overflow = ele.pstyle('text-overflow-wrap').value;
     let overflowAny = overflow === 'anywhere';
     let wrappedLines = [];
-    let wordsRegex = /[\s\u200b]+/;
-    let wordSeparator = overflowAny ? '' : ' ';
+    let separatorRegex = /[\s\u200b]+|$/g; // Include end of string to add last word
```

Comment:
Since I am using separator positions instead of split, and that I am acessing the previous word for each match, I need to include the end of the string as a match so that the last word is also taken into consideration